projects/driver_controls/inc/throttle.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+// Module for the throttle.
+// Requires Ads1015, and soft timer to be initialized.
+// To use the module, init the ADS1015 for the pedal and pass its Ads1015Storage to throttle_init,
+// along with the two channels the pedal is connected to. At any time calling throttle_get_position
+// will give the current position of the pedal.
+// Note that throttle_storage should persist across functions.
+#include <stdint.h>
+#include "ads1015.h"
+#include "soft_timer.h"
+#include "status.h"
+
+typedef enum {
+  THROTTLE_ZONE_BRAKE = 0,
+  THROTTLE_ZONE_COAST,
+  THROTTLE_ZONE_ACCEL,
+  NUM_THROTTLE_ZONES
+} ThrottleZone;
```

Comment:
You could probably use this enum to index into a lookup table or something to make your zone/position calculation a little neater.